manim/mobject/geometry.py

```diff
@@ -743,6 +742,54 @@ def __init__(self, n=6, **kwargs):
         Polygon.__init__(self, *vertices, **kwargs)
 
 
+class ArcPolygon(VMobject):
+    """
+    The ArcPolygon is what it says, a polygon, but made from arcs.
+    More versatile than the standard Polygon.
```

Comment:
just so it links to the class for quick access
```suggestion
    More versatile than the standard :class:`Polygon`.
```